app/middleware/rateLimiter.js

```diff
@@ -2,7 +2,7 @@ const redis = require('../../config/redis')
 const redisClient = redis.redisClient
 const moment = require('moment')
 const WINDOW_SIZE_IN_HOURS = 24
-const MAX_WINDOW_REQUEST_COUNT = process.env.NODE_ENV === 'testing' ? 20 : 100
+const MAX_WINDOW_REQUEST_COUNT = process.env.NODE_ENV === 'testing' ? 20 : 500
```

Comment:
ideally, 20 and 500 should be constant.

Reply:
That is assigned to MAX_WINDOW_REQUEST_COUNT" variable and that is already treated as a constant.
Wherever I will store the constant I need a variable to store the 20 and 500 into a variable.

Reply:
Can you please tell me what you want to be done with constants?